install_files/ansible-base/inventory

```diff
@@ -5,7 +5,7 @@
 #
 # NOTE: after the install is complete, you can re-provision with Ansible but you will need to:
 # 1. Set up HidServAuth in your torrc with the values from app-ssh-aths and mon-ssh-aths
-# 2. Replace the IP addresses here with the corresponding .onion addresses
+# 2. Replace the IP addresses here with the corresponding addresses
```

Comment:
Updated with an inverse grep so that we don't need to edit that comment. This still works even if the comment is not present.